bitnami/elasticsearch/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 19.4.0
+version: 19.4.1
```

Comment:
There is a version conflict. Update the version taking into account the current version of master.